samples/auth-user-sample/authUserFlow.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {OAuth2Client} = require('google-auth-library');
+const readline = require('readline');
```

Comment:
Gotcha. Thanks!

Reply:
If we had to pick one, I actually prefer the readline version. That way it works in every environment, including Cloud Shell.

In Python, we provide both, but it's a lot easier because it's built into the auth libs.

Reply:
I do prefer writing the sample to work in all environments. I think I'll go back to the readline version just so we can get something published, then I can see if it makes sense to have both examples